system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -805,6 +805,8 @@ impl pallet_session::Config for Runtime {
 	type Keys = SessionKeys;
 	type WeightInfo = weights::pallet_session::WeightInfo<Runtime>;
 	type DisablingStrategy = ();
+	type Currency = Balances;
+	type KeyDeposit = ();
```

Comment:
Same here please. 

TODO for me: make a script that calculates the list of validators who cannot set deposit  due to not having enough free balance. Post it in the validator lounge chats. 

TODO for you: make sure this is noted in release notes.

Reply:
https://github.com/acatangiu/runtimes/pull/13 you can use this which has both the report + right deposit amounts.

Reply:
The `pallet_sesion` in KAH and PAH doesn't need any deposits -- this is for collator selection, and it is already guarded by allowing only those who have submitted candidacy for collation to set keys (which should have an implicit deposit already attached to it). 

Only the `pallet_session` in RCs need deposit as they can no longer know if someone setting a deposit is a validator candidate or not.